eclair-core/src/test/scala/fr/acinq/eclair/json/JsonSerializersSpec.scala

```diff
@@ -151,6 +151,14 @@ class JsonSerializersSpec extends AnyFunSuite with Matchers {
     JsonSerializers.serialization.write(inputInfo)(JsonSerializers.formats) shouldBe """{"outPoint":"9f0b9c0ce92c175ca4e78acfd13a718099c73818b6d3140cfe6f04ec052b5b34:42","amountSatoshis":456651}"""
   }
 
+  test("channel flags serialization") {
+    //object ChannelFlags {
+    //  val AnnounceChannel = 0x01.toByte
+    //  val Empty = 0x00.toByte
+    //}
+    // val announceChannel: Boolean = (channelFlags & 0x01) != 0
+  }
```

Comment:
Looks like you're missing that test ;)

Reply:
Yeah, I realized later that I didn't need a custom serializer... Besides, the serialization is actually tested by the `backward compatibility older codecs (integrity)`.

Remove this draft in 8d8ee2f2f.